src/renderer/components/icon/icon.tsx

```diff
@@ -31,6 +30,8 @@ export interface IconProps extends React.HTMLAttributes<any>, TooltipDecoratorPr
 
 @withTooltip
 export class Icon extends React.PureComponent<IconProps> {
+  public ref?: HTMLElement;
```

Comment:
We can use `React.creareRef` here https://reactjs.org/docs/refs-and-the-dom.html

Hovewer, I don't see any behavior difference.

Reply:
I'll use createRef